cmd/kube-apiserver/app/options/options.go

```diff
@@ -128,7 +128,7 @@ func (s *ServerRunOptions) Flags() (fss apiserverflag.NamedFlagSets) {
 	s.Etcd.AddFlags(fss.FlagSet("etcd"))
 	s.SecureServing.AddFlags(fss.FlagSet("secure serving"))
 	s.InsecureServing.AddFlags(fss.FlagSet("insecure serving"))
-	s.InsecureServing.AddDeprecatedFlags(fss.FlagSet("insecure serving"))
+	s.InsecureServing.AddUnqualifiedFlags(fss.FlagSet("insecure serving")) // TODO: remove it until kops stops using `--address`
```

Comment:
I don't think we'll actually wait for kops.  When we're past the deprecation cycle time, we can remove it.

Reply:
We should formally deprecate insecure ports for all components in 1.12 and remove them all at once in 1.13.

Reply:
> We should formally deprecate insecure ports for all components in 1.12 and remove them all at once in 1.13.

Yes, I agree. Let's make that super clear in the release notes.